MANIFEST.in

```diff
@@ -0,0 +1 @@
+graft mechanisms
```

Comment:
```suggestion
include mechanisms/*.cti
```
(`graft` has the potential to pick up unwanted files, such as backups)

Reply:
bb15df6

Reply:
This change seems to break the examples.

Reply:
No it doesn't.  09f5666